### PR TITLE
docs(cn): translate guide/deploy.md

### PR DESCRIPTION
**Pull request by watonyweng**

### CODE REVIEW

**Comment by watonyweng** [comment]
@QC-L @ryanmoyo

File: guide/deploy.md [hunk 1]
```diff
@@ -243,7 +243,7 @@ $ heroku apps:create example
 $ heroku buildpacks:set https://github.com/heroku/heroku-buildpack-static.git
 ```
 
-6. Deploy your site:
+6. 部署您的站点。
```
Comment (meouwu3) [inline guide/deploy.md]:
```suggestion
6. 部署你的站点。
```

File: guide/deploy.md [hunk 2]
```diff
@@ -50,17 +50,17 @@ You may configure the port of the server py passing `--port` flag as an argument
 }
 ```
 
-Now the `docs:serve` method will launch the server at http://localhost:8080.
+现在执行 `docs: serve` 命令将在 http://localhost:8080 启动服务器。
```
Comment (meouwu3) [inline guide/deploy.md]:
```suggestion
现在执行 `docs:serve` 命令将在 http://localhost:8080 启动服务器。
```

File: guide/deploy.md [hunk 3]
```diff
@@ -50,17 +50,17 @@ You may configure the port of the server py passing `--port` flag as an argument
...
+## GitHub Pages {#github-pages}
 
-1. Set the correct `base` in `docs/.vitepress/config.js`.
+1. 在 `docs/.vitepress/config.js` 中设置正确的 `base`：
 
-   If you are deploying to `https://<USERNAME>.github.io/`, you can omit `base` as it defaults to `'/'`.
+   如果你部署到 `https://<USERNAME>.github.io/`，可以忽略 `base`，因为它默认为 `'/'`。
 
-   If you are deploying to `https://<USERNAME>.github.io/<REPO>/`, for example your repository is at `https://github.com/<USERNAME>/<REPO>`, then set `base` to `'/<REPO>/'`.
+   如果你部署到 `https://<USERNAME>.github.io/<REPO>/`，例如你的存储库是在 `https://github.com/<USERNAME>/<REPO>`，然后设置 `base` 为 `/<REPO>/`。
```
Comment (meouwu3) [inline guide/deploy.md:61]:
```suggestion
   如果你部署到 `https://<USERNAME>.github.io/<REPO>/`，例如你的存储库是在 `https://github.com/<USERNAME>/<REPO>`，然后设置 `base` 为 `'/<REPO>/'`。
```

File: guide/deploy.md [hunk 4]
```diff
@@ -253,14 +253,14 @@ $ git push heroku master
...
 ```
 
-## Vercel
+## Vercel {#vercel}
 
-To deploy your VitePress app with a [Vercel for Git](https://vercel.com/docs/git), make sure it has been pushed to a Git repository.
+要使用 [Vercel for Git](https://vercel.com/docs/git) 部署你的 VitePress 应用，请确保已将其推送到 Git 仓库。
 
-Go to https://vercel.com/import/git and import the project into Vercel using your Git of choice (GitHub, GitLab or BitBucket). Follow the wizard to select the project root with the project's `package.json` and override the build step using `yarn docs:build` or `npm run docs:build` and the output dir to be `./docs/.vitepress/dist`
+访问 https://vercel.com/import/git，使用你选择的 Git (GitHub，GitLab 或 BitBucket) 将项目导入到 Vercel 中。按照向导的指示，使用项目的 `package.json` 选择项目根目录，并使用 `yarn docs：build` 或 `npm run docs：build` 覆盖构建步骤，并将输出目录设置为 `./docs/.vitepress/dist`。
```
Comment (meouwu3) [inline guide/deploy.md]:
```suggestion
访问 https://vercel.com/import/git，使用你选择的 Git (GitHub, GitLab 或 BitBucket) 将项目导入到 Vercel 中。按照向导的指示，使用项目的 `package.json` 选择项目根目录，并使用 `yarn docs:build` 或 `npm run docs:build` 覆盖构建步骤，并将输出目录设置为 `./docs/.vitepress/dist`。
```

File: guide/deploy.md [hunk 5]
```diff
@@ -156,22 +156,22 @@ pages:
...
+## Google Firebase {#google-firebase}
 
-1. Make sure you have [firebase-tools](https://www.npmjs.com/package/firebase-tools) installed.
+1. 确保你安装了 [firebase-tools](https://www.npmjs.com/package/firebase-tools)：
 
-2. Create `firebase.json` and `.firebaserc` at the root of your project with the following content:
+2. 在你的工程根目录下创建 `firebase.json` 和 `.firebaserc`，内容如下：
 
-`firebase.json`:
+`firebase.json`：
```
Comment (meouwu3) [inline guide/deploy.md]:
```suggestion
`firebase.json`:
```

File: guide/deploy.md [hunk 6]
```diff
@@ -182,7 +182,7 @@ pages:
 }
 ```
 
-`.firebaserc`:
+`.firebaserc`：
```
Comment (meouwu3) [inline guide/deploy.md]:
```suggestion
`.firebaserc`:
```

File: guide/deploy.md [hunk 7]
```diff
@@ -192,43 +192,43 @@ pages:
...
+3. 运行 `yarn docs:build` 或 `npm run docs:build` 后，使用 `firebase deploy` 命令进行部署。
 
-## Surge
+## Surge {#surge}
 
-1. First install [surge](https://www.npmjs.com/package/surge), if you haven’t already.
+1. 如果尚未安装，请首先安装 [surge](https://www.npmjs.com/package/surge)：
 
-2. Run `yarn docs:build` or `npm run docs:build`.
+2. 运行 `yarn docs:build` 或 `npm run docs:build`：
```
Comment (meouwu3) [inline guide/deploy.md]:
```suggestion
2. 运行 `yarn docs:build` 或 `npm run docs:build`。
```

File: guide/deploy.md [hunk 8]
```diff
@@ -192,43 +192,43 @@ pages:
...
 ```
 
-3. After running `yarn docs:build` or `npm run docs:build`, deploy using the command `firebase deploy`.
+3. 运行 `yarn docs:build` 或 `npm run docs:build` 后，使用 `firebase deploy` 命令进行部署。
 
-## Surge
+## Surge {#surge}
 
-1. First install [surge](https://www.npmjs.com/package/surge), if you haven’t already.
+1. 如果尚未安装，请首先安装 [surge](https://www.npmjs.com/package/surge)：
```
Comment (meouwu3) [inline guide/deploy.md]:
```suggestion
1. 如果尚未安装，请首先安装 [surge](https://www.npmjs.com/package/surge)。
```

File: guide/deploy.md [hunk 9]
```diff
@@ -192,43 +192,43 @@ pages:
...
+3. 通过输入 `surge docs/.vitepress/dist` 部署到 surge。
 
-You can also deploy to a [custom domain](https://surge.sh/help/adding-a-custom-domain) by adding `surge docs/.vitepress/dist yourdomain.com`.
+你也可以通过添加 `surge docs/.vitepress/dist yourdomain.com` 部署到[自定义域名](https://surge.sh/help/adding-a-custom-domain)。
 
-## Heroku
+## Heroku {#heroku}
 
-1. Install [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli).
+1. 安装 [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli)：
```
Comment (meouwu3) [inline guide/deploy.md]:
```suggestion
1. 安装 [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli)。
```

File: guide/deploy.md [hunk 10]
```diff
@@ -192,43 +192,43 @@ pages:
...
 
-2. Create a Heroku account by [signing up](https://signup.heroku.com).
+2. 通过[注册](https://signup.heroku.com)来创建 Heroku 账号：
```
Comment (meouwu3) [inline guide/deploy.md]:
```suggestion
2. 通过[注册](https://signup.heroku.com)来创建 Heroku 账号。
```

File: guide/deploy.md [hunk 11]
```diff
@@ -192,43 +192,43 @@ pages:
...
 
 ```bash
 $ heroku login
 ```
 
-4. Create a file called `static.json` in the root of your project with the below content:
+4. 在你的工程根目录下创建一个名为 `static.json` 的文件，内容如下：
 
-`static.json`:
+`static.json`：
```
Comment (meouwu3) [inline guide/deploy.md]:
```suggestion
`static.json`:
```

File: guide/deploy.md [hunk 12]
```diff
@@ -253,14 +253,14 @@ $ git push heroku master
...
-To deploy your VitePress app with a [Vercel for Git](https://vercel.com/docs/git), make sure it has been pushed to a Git repository.
+要使用 [Vercel for Git](https://vercel.com/docs/git) 部署你的 VitePress 应用，请确保已将其推送到 Git 仓库。
 
-Go to https://vercel.com/import/git and import the project into Vercel using your Git of choice (GitHub, GitLab or BitBucket). Follow the wizard to select the project root with the project's `package.json` and override the build step using `yarn docs:build` or `npm run docs:build` and the output dir to be `./docs/.vitepress/dist`
+访问 https://vercel.com/import/git，使用你选择的 Git (GitHub，GitLab 或 BitBucket) 将项目导入到 Vercel 中。按照向导的指示，使用项目的 `package.json` 选择项目根目录，并使用 `yarn docs：build` 或 `npm run docs：build` 覆盖构建步骤，并将输出目录设置为 `./docs/.vitepress/dist`。
 
 ![Override Vercel Configuration](../images/vercel-configuration.png)
 
-After your project has been imported, all subsequent pushes to branches will generate Preview Deployments, and all changes made to the Production Branch (commonly "main") will result in a Production Deployment.
+导入项目后，所有后续到分支的推送都将生成预览部署，并且对生产分支所做的所有更改 (通常为“main”分支) 将导致生产部署。
```
Comment (meouwu3) [inline guide/deploy.md]:
```suggestion
导入项目后，所有后续到分支的推送都将生成预览部署，并且对生产分支所做的所有更改 (通常为 "main" 分支) 将导致生产部署。
```

Comment (meouwu3) [reply guide/deploy.md]: > 酌情采纳

**Review by eggx8 — COMMENTED** [review]
These are my suggestions, partially taken as appropriate.
You can use squash merge to avoid unnecessary information when merging PR.

这些是我的建议，部分酌情采纳。
合并 PR 的时候可以使用 squash & merge 避免多余的信息。

**Comment by watonyweng** [comment]
@KimYangOfCat

**Review by eggx8 — APPROVED** [review]

**Pull request merged**